tutorials/lab/rabbit.rst

```diff
@@ -18,6 +18,53 @@ creating jobspecs directly.)
 On the command line, set the ``.attributes.system.dw`` field by passing flags
 like ``-S dw="my_string"`` or ``--setattr=dw="my_string"``.
 
+The simplest way to use rabbits is to use preset DW strings that
+administrators may have configured on a system. For more flexibility,
+it may be desirable to write custom DW strings.
+
+
+Preset DW Directives
+--------------------
+
+Running
+
+.. code-block:: bash
+
+	flux config get rabbit.presets | jq .
+
+will display a mapping of the presets that have been configured on the
+current system. The keys of the mapping are the strings that you can
+pass to ``-S dw=...``, the values of the mapping are the actual DW directives
+they presets correspond to.
```

Comment:
Sorry if I am misunderstanding; `their` instead of `they` here?

Reply:
Good catch thanks! I think I wanted `the` actually (although `their` would also work). Fixed and force-pushed. Setting MWP